src/NHibernate.Test/Async/SessionBuilder/Fixture.cs

```diff
@@ -189,51 +189,6 @@ public async Task CanSetConnectionAsync()
 				FlushMode.Always, FlushMode.Auto, FlushMode.Commit, FlushMode.Manual);
 		}
 
-		[Test]
-		public async Task CanSetInterceptorAsync()
```

Comment:
Removed as it's not async (see https://github.com/maca88/AsyncGenerator/issues/116)